lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3418,12 +3424,12 @@ def test_get_ora2_responses_already_running(self):
             response = self.client.post(url, {})
 
         self.assertEqual(response.status_code, 400)
-        self.assertIn(already_running_status, response.content)
+        self.assertIn(already_running_status, response.content.decode('utf-8'))
 
     def test_get_student_progress_url(self):
         """ Test that progress_url is in the successful response. """
         url = reverse('get_student_progress_url', kwargs={'course_id': text_type(self.course.id)})
-        data = {'unique_student_identifier': self.students[0].email.encode("utf-8")}
+        data = {'unique_student_identifier': self.students[0].email}
```

Comment:
it added  `b` prefix like this with post data.
```b'robot'``` prefix with username and when it reaches inside view it returns some int value and user lookup db query fails.